src/styles/_catalog.scss

```
@@ -1,22 +1,37 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  $column-width: 200px;
+  $column-height: 300px;
 
-  max-width: 944px;
+  display: grid;
+  gap: 48px;
+  grid-template-columns: 200px;
+  place-items: center;
+
+  justify-content: center;
   margin: 0 auto;
+  padding-bottom: 48px;
 
   &__card {
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    width: 200px;
-    height: 300px;
+    display: grid;
+    grid-template-columns: #{$column-width};
+    grid-template-rows: #{$column-height};
+    place-items: center;
+    width: #{$column-width};
+    min-height: #{$column-height};
+
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 488px) {
+    grid-template-columns: repeat(2, #{$column-width});
```

can you use vars like that?

```suggestion
    grid-template-columns: repeat(2, $column-width);
```